textAdventure.js

```diff
@@ -0,0 +1,229 @@
+class Room {
+    constructor(name,description,items,north,east,south,west){
+    this.name=name //string
+    this.description=description //string
+    this.items=items //array?
+    this.north=north //room object
```

Comment:
good use of comments to explain what type of input is expected, but don't forget to update your comments as the code changes.  Your directions are no longer the objects, they are now strings.